sdk/go/common/util/gitutil/git_test.go

```diff
@@ -378,6 +378,22 @@ func TestTryGetVCSInfoFromSSHRemote(t *testing.T) {
 		{"svn:something.com/owner/repo", nil},
 		{"https://bitbucket.org/foo.git", nil},
 		{"git@github.foo.acme.bad-tld:owner-name/repo-name.git", nil},
+		{
```

Comment:
I'd like us to have a test setting the GITHUB_TOKEN for `github.enterprise.example.com`, similar to https://github.com/pulumi/pulumi/blob/master/sdk/go/common/util/gitutil/git_test.go#L457.